### PR TITLE
Add terminal_tutorial Ukrainian translation

### PR DESCRIPTION
**Pull request by dmytrovoytko**
Please, consider Ukrainian translation

### CODE REVIEW

File: terminal_tutorial.md [hunk 1]
```diff
@@ -83,8 +83,7 @@ You will make a copy of the card template inside the HTML file and customize it
 
 This tutorial is also available in [other languages](/translations/translation.md)
 
-| [English](/terminal-tutorial.md) | [Bangla](/translations/terminal_tutorial/terminal_tutorial.ben.md) |
-| :---: | :----: |
+| [English](/terminal_tutorial.md) | [Bangla](/translations/terminal_tutorial/terminal_tutorial.ben.md) | [Ukrainian](/translations/terminal_tutorial/terminal_tutorial.ukr.md) | :----: | :----: |
```
Comment (TenTraicion) [inline terminal_tutorial.md:86]: the table is broken

File: translations/translation.md [hunk 2]
```diff
@@ -36,7 +36,6 @@ All Translations regarding Project Documentations are welcome. There are certain
 
 ## Terminal Tutorial
 
-| 1   | [English](../terminal_tutorial.md) | [Bangla](terminal_tutorial/terminal_tutorial.ben.md) |
-| :---: | :---: | :---: |
+| [English](/terminal_tutorial.md) | [Bangla](/translations/terminal_tutorial/terminal_tutorial.ben.md) | [Ukrainian](/translations/terminal_tutorial/terminal_tutorial.ukr.md) | :----: | :----: |
```
Comment (TenTraicion) [inline translations/translation.md:39]: this one is also broken

**Review by TenTraicion — CHANGES_REQUESTED** [review]
@dmytrovoytko, Thanks for your contribution. Kindly fix the broken tables and we are good to go.

**Review by TenTraicion — APPROVED** [review]

**Comment by dmytrovoytko** [comment]
@TenTraicion fixed. Thank you for fast approval!

**Pull request merged**